api/db/populate-users.js

```diff
@@ -0,0 +1,133 @@
+#! /usr/bin/env node
+
+// TODO: This will all be in the general js to abide by DRY principal
+const userArgs = process.argv.slice(2);
+
+if (!userArgs[0].startsWith("mongodb")) {
+  console.log(
+    "Error: you need to specify a valid MongoDB URL as the first argument"
+  );
+
+  return;
+}
+
+const User = require("../models/User");
+const Interest = require("../models/Interest");
+
+const asynchronous = require("async");
+
+const mongoose = require("mongoose");
+const mongoDB = userArgs[0];
+const numberOfRecords = userArgs[1] || 100;
+mongoose.connect(mongoDB, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+  useCreateIndex: true,
+});
+mongoose.Promise = global.Promise;
+const db = mongoose.connection;
+db.on("error", console.error.bind(console, "MongoDB connection error:"));
+
+const { uniqueNamesGenerator, names } = require("unique-names-generator");
+const countries = require("country-region-data");
+const locales = ["en", "es"];
+const users = [];
+
+const createUser = async (userParams, callback) => {
+  const user = new User({
+    name: userParams.name,
+    email: userParams.email,
+    emailConfirmed: userParams.emailConfirmed,
+    emailConfirmedAt: userParams.emailConfirmedAt,
+    dateOfBirth: userParams.dateOfBirth,
+    country: userParams.country,
+    region: userParams.region,
+    preferredLocale: userParams.preferredLocale,
+    interests: userParams.interests,
+    admin: userParams.admin,
+  });
+  user.setPassword("password");
+  await user.save();
+
+  console.log(`New User: ${user}`);
+  users.push(user);
+  callback(null, user);
+};
+
+const getRandomDate = (startDate, endDate) =>
+  new Date(+startDate + Math.random() * (endDate - startDate));
+
+const createAdminUser = () => (seriesCallback) => {
+  createUser(
+    {
+      name: "Admin",
+      email: `admin@tisn.app`,
+      emailConfirmed: true,
+      emailConfirmedAt: new Date(),
+      country: "US",
+      region: "FL",
+      preferredLocale: "en",
+      dateOfBirth: new Date(),
```

Comment:
This won't be a valid `dateOfBirth`, as the user's age needs to be more than 13.